include/freerdp/client/audin.h

```diff
@@ -24,38 +24,28 @@
 
 #include <freerdp/channels/audin.h>
 #include <freerdp/freerdp.h>
+#include <freerdp/codec/audio.h>
 
 /**
  * Subsystem Interface
  */
 
-typedef UINT (*AudinReceive) (const BYTE* data, int size, void* userData);
-
-typedef struct audin_format audinFormat;
-struct audin_format
-{
-	UINT16 wFormatTag;
-	UINT16 nChannels;
-	UINT32 nSamplesPerSec;
-	UINT16 nBlockAlign;
-	UINT16 wBitsPerSample;
-	UINT16 cbSize;
-	BYTE* data;
-};
+typedef UINT(*AudinReceive)(const AUDIO_FORMAT* format,
+                            const BYTE* data, size_t size, void* userData);
 
 typedef struct _IAudinDevice IAudinDevice;
 struct _IAudinDevice
 {
-	UINT (*Open) (IAudinDevice* devplugin, AudinReceive receive, void* userData);
-	BOOL (*FormatSupported) (IAudinDevice* devplugin, audinFormat* format);
-	UINT (*SetFormat) (IAudinDevice* devplugin, audinFormat* format, UINT32 FramesPerPacket);
-	UINT (*Close) (IAudinDevice* devplugin);
-	UINT (*Free) (IAudinDevice* devplugin);
+	UINT(*Open)(IAudinDevice* devplugin, AudinReceive receive, void* userData);
+	BOOL (*FormatSupported)(IAudinDevice* devplugin, const AUDIO_FORMAT* format);
+	UINT(*SetFormat)(IAudinDevice* devplugin, const AUDIO_FORMAT* format, UINT32 FramesPerPacket);
+	UINT(*Close)(IAudinDevice* devplugin);
```

Comment:
missing space after return type ?

Reply:
Ok, will check formatting, looks like I forgot that before commit.